example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
 export "FLUTTER_ROOT=C:\flutter"
-export "FLUTTER_APPLICATION_PATH=E:\flutter_projects\Animated-Text-Kit\example"
+export "FLUTTER_APPLICATION_PATH=C:\Users\Legion\AndroidStudioProjects\Animated-Text-Kit\example"
```

Comment:
@AadumKhor, please add this file to `.gitignore` and remove it from the git history.